README.md

```diff
@@ -1,8 +1,8 @@
 # UID2 Integration Examples
 
-The [UID2 framework](https://github.com/UnifiedID2/uid2docs/tree/main) enables publishers to integrate in either of the following ways:
+The [UID2 framework](https://unifiedid.com/docs/intro) enables publishers to integrate in either of the following ways:
 
-- Via the standard integration workflow, using the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md) (also known as the UID2 SDK).
+- Via the standard integration workflow, using the [Client-Side Identity JavaScript SDK](https://unifiedid.com/docs/sdks/client-side-identity) (also known as the UID2 SDK).
```

Comment:
Change title to Client-Side JavaScript SDK. We took Identity out of the title. Please update any instances of this. The V2 is still in question per thread with Andrei but I believe he'll say we don't need it. But we definitely don't need Identity.